weave/weave_client.py

```diff
@@ -442,6 +443,28 @@ def create_call(
         inputs: dict,
         attributes: dict = {},
     ) -> Call:
+        if parent is None:
+            parent = run_context.get_current_run()
+        call = self.create_call_outside_execution(op, parent, inputs, attributes)
+        run_context.push_call(call)
+        return call
+
+    @trace_sentry.global_trace_sentry.watch()
+    def create_call_outside_execution(
+        self,
+        op: Union[str, Op],
+        parent: Optional[Call],
+        inputs: dict,
+        attributes: dict = {},
```

Comment:
Should avoid using a mutable object as a default value

Reply:
fixed